### PR TITLE
add docs for The 1:1 Hour

### PR DESCRIPTION
**Pull request by parispittman**
adding the documentation for The 1:1 Hour - a mentoring program. The last link will be dead until I upload the mentoring guidelines. 

/sig contributor-experience
/hold

### CODE REVIEW

File: mentoring/the1-on-1hour.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+* AMA session (ask anything!)
+
+We will attempt to match you with the closest mentor of your area of interested or activity. 
+
+## Sign up!
+[Fill out this form](https://goo.gl/forms/9WllkPFTRB999vcc2) and we'll be in touch! 
+
+## Mentors
+
+If you are interested in signing up to be a mentor for this, please see the [mentor guidelines](mentorguidelines.md)
```
Comment (tpepper) [inline mentoring/the1-on-1hour.md]: Period police.  ;0

File: mentoring/the1-on-1hour.md [hunk 2]
```diff
@@ -0,0 +1,29 @@
+The 1:1 Hour
+=======
+*WIP*
+
+One mentor. One mentee. One Hour.
+
+In this mentorship opportunity, you'll be paired with someone in the ecosystem that can help drive your upstream Kubernetes experience forward with a quick burst of information. The mentee controls the content that will be served during that hour. This is open for new AND current contributors of all [levels](/community-membership.md).
+
+Activites that you can select from include:
```
Comment (cblecker) [inline mentoring/the1-on-1hour.md]:
```
W0413 05:15:40.901] mentoring/the1-on-1hour.md:9:0: "Activites" is a misspelling of "Activities"
```

**Comment by parispittman** [comment]
@tpepper @justaugustus - let me know your thoughts on the changes to this one, too.

File: mentoring/the1-on-1hour.md [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+* AMA session (ask anything!)
+
+We will attempt to match you with the closest mentor of your area of interested or activity. 
+
+## Sign up!
+[Fill out this form](https://goo.gl/forms/9WllkPFTRB999vcc2) and we'll be in touch! 
+
+## Mentors
+
+If you are interested in signing up to be a mentor for this, please see the [mentor guidelines](mentorguidelines.md).
```
Comment (tpepper) [inline mentoring/the1-on-1hour.md]: This link is broken when the other PR is not merged, but that PR references this one.  SOoo PR 2041 and 2042 should be merged in close succession or one of them combined into the other so they merge at the same time.

**Comment by cblecker** [comment]
/approve

🥇 🎈

**Pull request merged**